docs/sources/datasources/graphite/template-variables/index.md

```diff
@@ -37,122 +37,155 @@ refs:
 # Graphite template variables
 
 Instead of hard-coding details such as server, application, and sensor names in metric queries, you can use variables.
-Grafana lists these variables in dropdown select boxes at the top of the dashboard to help you change the data displayed in your dashboard.
+Grafana lists these variables in drop-down selection boxes at the top of the dashboard to help you change the data displayed in your dashboard.
 Grafana refers to such variables as template variables.
 
 For an introduction to templating and template variables, refer to the [Templating](ref:variables) and [Add and manage variables](ref:add-template-variables) documentation.
 
-## Select a query type
+To view an example templated dashboard, refer to [Graphite Templated Nested dashboard](https://play.grafana.org/d/cvDFGseGz/graphite-templated-nested).
+
+## Use query variables
+
+With Graphite, the most commonly supported variables are query variables.
```

Comment:
The only type of variables you can make with a Graphite data source are query variables. Adhoc filters aren't supported.